module-utils/utility/Utils.hpp

```diff
@@ -158,6 +158,11 @@ namespace utils
         if (value.empty()) {
             return {};
         }
+
+        if constexpr (std::is_same<uint8_t, T>::value || std::is_same<int8_t, T>::value) {
```

Comment:
The same condition is checked twice.

Reply:
actually it's unsigned and a singed check :)

Reply:
Oh, my bad.